cdap-docs/admin-manual/source/operations/tx-maintenance.rst

```diff
@@ -35,11 +35,11 @@ To prune the invalid list manually, follow these steps:
   Pick the minimum time across all region servers. In this case, ``1440202895873``.
 
 - Find the minimum prune time across all queues by running the tool ``SimpleHBaseQueueDebugger``.
-  Note that authorization is disabled when running the SimpleHBaseQueueDebugger, so that cdap can read all
-  users' tables.
-  This should print lines such as::
+  Note that authorization is disabled when running the ``SimpleHBaseQueueDebugger`` so that CDAP can read all
+  users' tables. This should print lines such as::
 
     $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.SimpleHBaseQueueDebugger
+    
```

Comment:
Well, maybe its just a nitpick, but then it deviates from how the lines actually look on the user's terminal, which we imply its the same.